src/amo/components/GuidesAddonCard/index.js

```diff
@@ -0,0 +1,225 @@
+/* @flow */
+import * as React from 'react';
+import { compose } from 'redux';
+import { connect } from 'react-redux';
+
+import AddonTitle from 'amo/components/AddonTitle';
+import AddonCompatibilityError from 'amo/components/AddonCompatibilityError';
+import { makeQueryStringWithUTM } from 'amo/utils';
+import AMInstallButton from 'core/components/AMInstallButton';
+import {
+  INCOMPATIBLE_NOT_FIREFOX,
+  INSTALL_SOURCE_GUIDES_PAGE,
+  UNKNOWN,
+} from 'core/constants';
+import { getAddonIconUrl } from 'core/imageUtils';
+import { withInstallHelpers } from 'core/installAddon';
+import translate from 'core/i18n/translate';
+import { getVersionById } from 'core/reducers/versions';
+import { getErrorMessage } from 'core/utils/addons';
+import { getClientCompatibility } from 'core/utils/compatibility';
+import Card from 'ui/components/Card';
+import Icon from 'ui/components/Icon';
+import Button from 'ui/components/Button';
+import Notice from 'ui/components/Notice';
+import type { AddonType } from 'core/types/addons';
+import type { AddonVersionType } from 'core/reducers/versions';
+import type { AppState } from 'amo/store';
+import type { UserAgentInfoType } from 'core/reducers/api';
+import type { I18nType } from 'core/types/i18n';
+import type { LocationType } from 'core/types/router';
+import type { WithInstallHelpersInjectedProps } from 'core/installAddon';
+
+import './styles.scss';
+
+type Props = {
+  addon: AddonType,
+  addonCustomText: string,
+  staffPick?: boolean,
+};
+
+type InternalProps = {
+  ...Props,
+  ...WithInstallHelpersInjectedProps,
+  _getClientCompatibility: typeof getClientCompatibility,
+  addon: AddonType | null,
+  clientApp: string | null,
+  currentVersion: AddonVersionType | null,
+  defaultInstallSource: string,
+  installError: string | null,
+  installStatus: string,
+  i18n: I18nType,
+  location: LocationType,
```

Comment:
I think you can remove this prop type because `WithInstallHelpersInjectedProps` already defines it, right?

Reply:
I don't see it there currently (https://github.com/mozilla/addons-frontend/blob/master/src/core/installAddon.js#L245)

Reply:
Oh, huh, it wouldn't be in injected props (because it doesn't get injected) but it is a required prop. I see it [WithInstallHelpersProps](https://github.com/mozilla/addons-frontend/blob/master/src/core/installAddon.js#L216) but maybe that won't help you. OK, I guess you need to define it like you have done.